app/models/company.rb

```diff
@@ -0,0 +1,28 @@
+class Company < ApplicationRecord
+  VALID_PHONE = /\A\d{10}\z/.freeze
```

Comment:
add frozen string literal at the top of the file and remove .freeze on constants